data/data/azure/variables-azure.tf

```diff
@@ -158,3 +158,11 @@ This determined whether User defined routing will be used for egress to Internet
 When false, Standard LB will be used for egress to the Internet.
 EOF
 }
+
+variable "azure_hypervgeneration_version" {
+  type = string
+  description = <<EOF
+This determines the HyperVGeneration disk type to use. This will be used to also
```

Comment:
"also"? Isn't that the only use. I don't know that we need to include details on how the variable is being used anyway.